src/components/Table/Table.story.jsx

```diff
@@ -700,7 +700,7 @@ storiesOf('Watson IoT/Table', module)
             ...initialState.options,
             hasResize: true,
             hasFilter: select('hasFilter', ['onKeyPress', 'onEnterAndBlur'], 'onKeyPress'),
-            wrapCellText: select('wrapCellText', selectTextWrapping, 'always'),
+            wrapCellText: select('wrapCellText', selectTextWrapping, 'alwaysTruncate'),
```

Comment:
wouldn't we want all options in this list so that we can see how always, never, auto and alwaysTruncate behaves?

Reply:
Yes indeed, and you have all options defined in the `selectTextWrapping` variable. The `alwaysTruncate` is just the selected default for this story.